src/docs/software/using/R.md

```diff
@@ -202,6 +202,57 @@ Loading required package: parallel
 >
 ```
 
+#### Package dependencies
+
+Sometimes when installing R packages, other software is needed for the installation and/or compilation.  For example the package `sf`.  If you try to install it:
+
+```
+install.packages("sf", repos = "http://cran.us.r-project.org")
+```
+
+You will see the error:
+
+```
+Configuration failed because libudunits2.so was not found. Try installing:...
+```
+
+and:
+
+```
+configure: error: gdal-config not found or not executable.
+```
+
+This is because `sf` needs udunits and gdal (and a few others) in order to compile and install successfully.  Fortunately we already have udunits and gdal installed on Sherlock as modules.  When you see  "not found" or  "not executable." errors you should always search our modules with module spider first:
+
+```
+module spider udunits
+```
+
+```
```

Comment:
This one I'm not sure if it would look better without formatting. Maybe preview locally first with and without.

```suggestion
```bash
```